lighthouse-logger/index.js

```diff
@@ -24,6 +66,7 @@ const colors = {
 };
 
 // whitelist non-red/yellow colors for debug()
+// @ts-ignore
```

Comment:
combine comments (and maybe add something about it being non-public to explain the ts-ignore)